buildSrc/src/main/java/org/elasticsearch/gradle/internal/test/rest/transform/match/ReplaceMatch.java

```diff
@@ -21,18 +21,21 @@
  */
 public class ReplaceMatch implements RestTestTransformByParentObject {
     private final String replaceKey;
+    private final String newKeyName;
```

Comment:
This needs to be exposed as an `@Input`. Adding a public getter for this field would also remove the need to override it in `ReplaceKeyInMatch`.